services/catarse.js/legacy/src/vms/reward-vm.js

```diff
@@ -79,7 +79,12 @@ const selectReward = reward => () => {
         if (reward.id) {
             contributionValue(h.applyMonetaryMask(`${reward.minimum_value},00`));
         } else { // no reward
-            contributionValue(h.applyMonetaryMask('$10,00'));
+            if (contributionValue() === '10,00' || 
+                contributionValue() === '' || 
```

Comment:
These 4 next conditions are false in JS so you could just something like ```!contributionValue()``` instead.